### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by Djwszone** [community/community#64159] [New to GitHub]
Hello GitHub, I am seeking somebody who is willing to teach me how to properly code and hack. Now i dont like using that word so if someone can also tell me what it means to properly hack that would also be great

**Comment by HeavySnowJakarta** [comment]
As I understand, there are two things we can do with GitHub: getting others' codes, and contributing our own codes. If you don't know how to use GitHub to contribute, there are many tutorials on the Internet which can teach you how to use Git and how to start an issue or a pull request. Then, you can explore the projects of GitHub. There are so many kinds of projects on GitHub that I think you will find one that you are interested in. Understand what the project does, read the codes and think how those works, and start your contribution by trying to progress the codes if you can or writting the doc. That's how I understand "code and hack".
